### PR TITLE
Remove some unnecessary links

### PR DESCRIPTION
**Pull request by oscarkraemer**
The pouta-adding-images was visible in plain text for docs readers.

### CODE REVIEW

**Comment by joonas-somero** [comment]
The removed links are still referenced in the text, such as pictured here:

image

**Comment by JMuff22** [comment]
@oscarkraemer should the terms of use link be re-added as well? It is used on line 363 of `vm-flavors-and-billing.md` for example. 

Also optimization server (Line 360) also has a broken link now due to the remove lines from the bottom of the file.

**Comment by oscarkraemer** [comment]
Well this have been annoying :) hopefully it is better now.

File: docs/cloud/pouta/vm-flavors-and-billing.md [hunk 1]
```diff
@@ -510,6 +508,4 @@ but they will be removed at some point in the near future.
   [CSC computing environment articles]: https://research.csc.fi/computing
   [command line instructions]: command-line-tools.md
   [TensorFlow]: https://www.tensorflow.org
-  [Optimization service]: https://research.csc.fi/optimization-service
```
Comment (JMuff22) [inline docs/cloud/pouta/vm-flavors-and-billing.md:513]: Here is the optimization service removed link. It is in the text at line 360.

**Review by joonas-somero — APPROVED** [review]

**Pull request merged**